src/app/InteractionModelEngine.cpp

```diff
@@ -554,6 +554,14 @@ Protocols::InteractionModel::Status InteractionModelEngine::OnReadInitialRequest
                 return Status::InvalidAction;
             }
 
+            if (requestedAttributePathCount == 0 && requestedEventPathCount == 0)
```

Comment:
Given that:

1) Our current behavior matches the spec.
2) The spec change has not yet been accepted.

we should not change behavior here unless and until the spec either changes or is at least slated to change.